TeamCode/src/main/java/org/firstinspires/ftc/teamcode/lib/RobotCentricMechanumDrive.java

```diff
@@ -58,7 +59,10 @@ public RobotCentricMechanumDrive(HardwareMap hardwareMap, Direction motorFrontLe
     }
 
     public void drive(Gamepad gamepad, Telemetry telemetry) {
-        double speedFactor = gamepad.left_trigger * speedFactorMultiplier;
+        double speedFactor = gamepad.left_trigger * speedFactorMultiplier_leftTrigger;
+        if (gamepad.right_trigger != 0.0) {
+            speedFactor = gamepad.right_trigger * speedFactorMultiplier_rightTrigger;
```

Comment:
I think we might want to reconsider having a single speed multiplier that's the product (times) of two triggers as this seems to do.

Maybe we can have a baseline (effectuating the way 3231 has speedLimiter = 0.5), and one trigger slows down, while the other one speeds up.

I think in our next group session would be great to go over, with the students, how to express the ideal UI (as defined by students) in code.